settings.gradle.kts

```diff
@@ -8,3 +10,22 @@ include(
     "samples:gradle-export-api",
     "test_projects:android"
 )
+
+plugins {
+    id("com.gradle.enterprise") version "3.4.1"
+}
+
+gradleEnterprise {
+    buildScan {
+        termsOfServiceUrl = "https://gradle.com/terms-of-service"
+        termsOfServiceAgree = "yes"
```

Comment:
you should use `publishAlwaysIf { isCi }`. `--scan` loads the gradle enterprise in different mechanism.

Reply:
> --scan loads the gradle enterprise in different mechanism.

Could you explain?

Reply:
`--scan` loads the gradle enterprise plugin that is built in to the current Gradle build tool version whereas `publishAlwaysIf` will use the version configured in the `settings.gradle` file.

Reply:
I see, thanks!